client/ciaos/client.py

```diff
@@ -0,0 +1,27 @@
+import requests
+
+
+def save(API_URL, key, value):
+    try:
+        data = {"key": key, "encoded_content": value}
+        response = requests.post(f"{API_URL}/upload/", data=data)
```

Comment:
Ideally this response should be returned. If in the future if the key is generated by us or any success/failure message the library user want's to know how will he know?

Reply:
isnt it handled by the code
 if response.status_code == 200:
            print("Update successful:", response.json())
        else:
            raise requests.HTTPError(response.text)

    except requests.HTTPError as e:
        print("Error:", e)

Reply:
You are the printing upon 200 where are you returning?

Reply:
ah yes i gotta modify except requests.HTTPError as e: print("Error:", e)